website/docs/getting-started/tutorial.md

```diff
@@ -65,7 +59,7 @@ For reliable results we use version 2.1.18 (replace `[mime-types-dir]` with the
 ```shell
 git clone https://github.com/jshttp/mime-types.git [mime-types-dir]
 cd [mime-types-dir]
-git checkout 2.1.18
+git checkout 2.1.35
```

Comment:
Note that it might make sense to stick to the older 18 patch-level for comparability to some "mime-types-2.1.18" test assets we have in our code base, like https://github.com/oss-review-toolkit/ort/blob/main/scanner/src/test/assets/scancode-3.0.2_mime-types-2.1.18.json.

Reply:
> I updated because using 2.1.18 resulted in tons of issues due to outdated dependencies.

What kind of issues exactly? Do you mean security vulnerabilities? However, I believe we do not even run the advisor yet as part of the tutorial... see https://github.com/oss-review-toolkit/ort/issues/4219.